CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Changed
 * Updated [ARIA S1 GUNW Product Guide](https://hyp3-docs.asf.alaska.edu/guides/gunw_product_guide.md) to include information about coverage thresholds, polarization limitations, and ingesting On-Demand products to the ARIA-S1-GUNW archive after processing
```

Comment:
We've removing this content due to a change in project priorities. The changelog entry may well change again before you're ready to merge this content, so it would be worth double-checking at the time before merging. 
```suggestion
```